docs/en/advanced_guides/conventions.md

```diff
@@ -2,6 +2,38 @@
 
 Please check the following conventions if you would like to modify MMDetection as your own project.
 
+## Image pipeline input shape
+
+In OpenMMLab 2.0, the image pipeline input shape is `(width, height)`, the shape of the pipeline output is `(height, width)` and the shape is also always `(height, width)` in model. The common fields are as follows:
+
+- img_shape: (height, width)
+- ori_shape: (height, width)
+- pad_shape: (height, width)
+- batch_input_shape: (height, width)
+
+As an example, the initialization parameters of `Mosaic` are as follows:
```

Comment:
On the contrary, the argument that build each pipeline is in the order of `(width, height)`, for example, the initialization arguments of `Mosaic` are as below: